.github/workflows/testing.yml

```diff
@@ -0,0 +1,19 @@
+name: Testing Workflow
+
+on: [push]
```

Comment:
```suggestion
on:
  push:
  pull_request:
    branches: [ main ]
```

Reply:
I tried that but still no luck. @sudo-bmitch

Reply:
At this point, I'm out of ideas other than the maintainers checking things on their side or merging and see if that helps.

Reply:
Maybe they have to merge this and then subsequent PRs will be impacted properly???